### DISCUSSION TITLE
Stand up Monday May 15th

### DISCUSSION THREAD

**Discussion by CristianCazares** [Bugs-io/TC3001B#77] [Main Stand Ups]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
- 

**What are your priorities for the next 24 hours?**:
- 

**Blockers**:
- 

**Shoutouts**:
- @username for x
```

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
- Upload button connected to API

**What are your priorities for the next 24 hours?**:
- Merge al the pr's, connect to deployed backend and deploy advances to vercel

**Blockers**:
- My Cloud Account is blocked xd

**Comment by Jacobgtz024** [comment]
**What did you achieve in the last 24 hours?**:
- Plot some basic data

**What are your priorities for the next 24 hours?**:
- Send the data through an endpoint and try nivo out
- Know what to plot

**Blockers**:
- Nohay

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
-  Deployed dataservice and backend

**What are your priorities for the next 24 hours?**:
- Started on working on plotting tasks

**Blockers**:
-  Don't know what to do

**Shoutouts**:
- @CristianCazares for going to the gym
